src/translations/nb/profile.json

```diff
@@ -52,6 +52,7 @@
     "faculty": "",
     "institute": "",
     "institution": "Institusjon",
+    "no_institutions_found": "Ingen institusjoner funnet",
```

Comment:
denne er ikke på samme linje som for `en`, så vi har en mismatch på de filene... men det er vel en egen sak

Reply:
nei, tror det mangler noen translations der. kan ta det i en egen PR